src/Clipper.jl

```diff
@@ -534,6 +542,10 @@ function Base.length(c::__ClipperPolyTree)
     @cxx c->Total()
 end
 
+function child_count(c::__ClipperPolyTree)
+    @cxx c->ChildCount()
+end
+
```

Comment:
This needs tests. 

Apparently the PolyTree is just an array of PolyNodes: https://github.com/Voxel8/Clipper.jl/blob/6a3cce9ea409d7b4977024e3bbee1830955cb205/src/clipper_cpp.jl#L138

It would be convenient if we could define `getindex` too.